pkg/server/plugin/upstreamca/disk/disk_test.go

```diff
@@ -116,6 +119,21 @@ func (s *DiskSuite) TestSubmitValidCSR() {
 		require.Len(upstreamTrustBundle, 1)
 		require.Equal("spiffe://local", certURI(upstreamTrustBundle[0]))
 	}
+
+	for _, csrFile := range csrFiles {
+		testCSR(csrFile)
+	}
+
+	// Modify the cert and key file paths. The CSR will still be
+	// signed by the cached upstreamCA.
+	s.rawPlugin.mtx.Lock()
```

Comment:
It's too bad the test loads files from the source, otherwise we could just delete them from underneath the plugin instead of reaching into internals to force a failure. This is good enough for now though.